CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ### CHANGELOG
 
+#### 4.2.0
+
```

Comment:
You need to add this in `4.1.2` yet to be released version, Sorry for the confusion. Ideally, it should be 4.1-SNAPSHOT

Reply:
Ah, sorry about this, I should have asked. Updating..